### PR TITLE
Horizontal bar chart unit test cases

### PR DESCRIPTION
**Pull request by v-baambati**
1. Added unit test cases
2. No change in test coverage as most of the functions covered in component tests.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f90a7d66c47ffca2e307fb396d87540e0c1f7a1e'>f90a7d66c47ffca2e307fb396d87540e0c1f7a1e</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Comment by srmukher** [comment]
Add proper PR description along with screenshots as applicable.

File: packages/react-charting/docs/TestPlans/HorizontalBarChart/UnitTests.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+|                         - Without chartDataMode                         |                             Should return proper axis data without chartDataMode defined                             |      RTL      |
+|                      - With default chartDataMode                       |                              Should return proper axis data with default chartDataMode                               |      RTL      |
+|                      - With fraction chatDataMode                       |                              Should return proper axis data with fraction chartDataMode                              |      RTL      |
+|                     - With percentage chatDataMode                      |                             Should return proper axis data with percentage chartDataMode                             |      RTL      |
+|                       Test 2: Get Chat Data Text                        |                                                                                                                      |               |
```
Comment (AtishayMsft) [inline packages/react-charting/docs/TestPlans/HorizontalBarChart/UnitTests.md:10]: rename to chart

File: packages/react-charting/docs/TestPlans/HorizontalBarChart/UnitTests.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+|                           Test 4: Create Bars                           |                                                                                                                      |               |
+|                        - With proper chart data                         |                                           Should return bar count properly                                           |      RTL      |
+|                     - With horizontalbarchart data                      |                                           Should return bar width properly                                           |      RTL      |
+|                         - With barHeight as “x”                         |                                           Should return bar heigh properly                                           |      RTL      |
+|                         Test 5: Get Area Label                          |                                                                                                                      |               |
```
Comment (AtishayMsft) [inline packages/react-charting/docs/TestPlans/HorizontalBarChart/UnitTests.md:20]: aria label

**Review by AtishayMsft — APPROVED** [review]
Approved. Need to resolve the 2 typos in the md file before merging.

**Comment by v-baambati** [comment]
updated.

**Pull request merged**